src/backends/torch/native/templates/crnn.hpp

```diff
@@ -31,59 +31,192 @@
 
 namespace dd
 {
+
+  class VGGFeatImpl : public torch::nn::Cloneable<VGGFeatImpl>
```

Comment:
Is this used somewhere ?

Reply:
I did not implement it, I can remove it for now